src/components/layout/Signup.module.css

```diff
@@ -56,7 +56,7 @@
   white-space: nowrap;
 }
 
-.input {
+.inputField {
   width: 100%;
   padding: 10px;
   border: 1px solid #ccc;
```

Comment:
em here too

Reply:
John worked on all the design part. I will check with him.